e2e/test/scenarios/metrics/metrics-editing.cy.spec.js

```diff
@@ -283,21 +261,14 @@ describe("scenarios > metrics > editing", () => {
       verifyScalarValue("4,939");
     });
 
-    it("should create a metric based on a multi-stage metric", () => {
-      createQuestion(ORDERS_MULTI_STAGE_METRIC);
+    it("should not allow to create a multi-stage metric", () => {
       startNewMetric();
       entityPickerModal().within(() => {
-        entityPickerModalTab("Metrics").click();
-        cy.findByText(ORDERS_MULTI_STAGE_METRIC.name).click();
-      });
-      addDateBetweenFilter({
-        columnName: "Created At: Month",
-        minValue: "May 7, 2020",
-        maxValue: "October 20, 2022",
+        entityPickerModalTab("Models").click();
+        cy.findByText("Orders Model").click();
       });
-      saveMetric();
-      runQuery();
-      verifyScalarValue("6");
+      addAggregation({ operatorName: "Count of rows" });
+      getActionButton("Summarize").should("not.exist");
```

Comment:
Nit: we can also assert that there are no join and no custom column buttons in/after the summarize step.

Reply:
This is done in a unit test

Reply:
I.e. the unit test checks that there are only "sort" and "limit"

Reply:
The unit test does not assert `steps.length`

Reply:
stages are not appended via steps, only actions